master/buildbot/status/web/mybuilds.py

```diff
@@ -11,18 +11,35 @@ class MybuildsResource(HtmlResource):
     @defer.inlineCallbacks
     def content(self, req, cxt):
         master = self.getBuildmaster(req)
+        user_id = cxt['authz'].getUserInfo(cxt['authz'].getUsername(req))['uid']
+
+        cxt['builds'] = yield self.prepare_builds(master, user_id)
+        cxt['days_count'] = master.config.myBuildDaysCount
+        template = req.site.buildbot_service.templates.get_template("mybuilds.html")
+        template.autoescape = True
+        defer.returnValue(template.render(**cxt))
+
+    @staticmethod
+    @defer.inlineCallbacks
+    def prepare_builds(master, user_id):
```

Comment:
Please add some docstring to public functions/methods

Reply:
Docstrings added for function in master.buildbot.util.build